server/src/databases/cosmosClient.test.ts

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Containers, ContainerResponse, ContainerRequest, Databases, DatabaseResponse, Items } from '@azure/cosmos';
+import CosmosClient from './cosmosClient';
+import { getServerConfig } from '../utils/getConfig';
+
+describe('Test cosmosClient', () => {
+  afterEach(() => {
+    delete process.env.VV_COSMOS_DB_CONNECTION_STRING;
+  });
+
+  test('Initialize cosmosClient when VV_COSMOS_DB_CONNECTION_STRING present.', async () => {
+    const expectedEndpoint = 'https://testinghost/';
+    const expectedAccountKey = 'testAccountKey';
+    process.env.VV_COSMOS_DB_CONNECTION_STRING = `AccountEndpoint=${expectedEndpoint};AccountKey=${expectedAccountKey}`;
+
+    const config = getServerConfig();
```

Comment:
I think we can just mock the config and not call external functions so that the unit tests are isolated.